mne/viz/_3d.py

```diff
@@ -1931,7 +1931,7 @@ def _plot_dipole(ax, data, points, idx, dipole, gridx, gridy, ori, coord_frame,
                 zdir='y', cmap='gray', zorder=0, alpha=.5)
 
     plt.suptitle('Dipole %s, Time: %.3fs, GOF: %.1f, Amplitude: %.1fnAm\n' % (
-        idx, dipole.times[idx], dipole.gof[idx], dipole.amplitude[idx] * 1e9) +
+        idx + 1, dipole.times[idx], dipole.gof[idx], dipole.amplitude[idx] * 1e9) +
```

Comment:
@agramfort @Eric89GXL another option would be to use # as in ICA which seems to indicate index semantics in our plots (E.g. ```ICA #000```)